source/boards/get-started-with-boards.rst

```diff
@@ -0,0 +1,78 @@
+Get Started with Mattermost Boards
+==================================
+
+Access Mattermost Boards by selecting **Boards** from the **Product list** |product-list| icon.
```

Comment:
This page may need to be added to the TOC.
The icon doesn't seem to render properly on preview.